compiler/rustc_builtin_macros/src/format.rs

```diff
@@ -995,8 +995,9 @@ pub fn expand_preparsed_format_args(
             e.note(&note);
         }
         if let Some((label, span)) = err.secondary_label {
-            let sp = fmt_span.from_inner(span);
-            e.span_label(sp, label);
+            // FIXME(compiler-errors): Should we just suppress this secondary_label?
+            let sp2 = if efmt_kind_is_lit { fmt_span.from_inner(span) } else { fmt_span };
+            e.span_label(sp2, label);
```

Comment:
`sp2` looks like a leftover temporary change. If so:
```suggestion
            let sp = if efmt_kind_is_lit { fmt_span.from_inner(span) } else { fmt_span };
            e.span_label(sp, label);
```